src/nix_auto_follow/cli.py

```diff
@@ -64,6 +64,11 @@ def get_url(self) -> str:
                 return f"github:{original['owner']}/{original['repo']}{rev_or_ref}"
             case "gitlab":
                 return f"gitlab:{original['owner']}/{original['repo']}{rev_or_ref}"
+            case "sourcehut":
```

Comment:
If you look at the test file should be pretty straightforward to add a case for sourcehut; could you do that please?

Reply:
I'll add a test file for sourcehut. Thank you!